src/sinks/amqp/config.rs

```diff
@@ -26,6 +54,9 @@ pub struct AmqpSinkConfig {
     /// Template used to generate a routing key which corresponds to a queue binding.
     pub(crate) routing_key: Option<Template>,
 
+    /// AMQP Properties for the messages.
```

Comment:
(same question about capitalization)

Reply:
Changed for the same phrasing suggested before "AMQP message properties"